app/controllers/api/products_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Api::ProductsController < ApplicationController
+  def index
+    products = Product.all
+    render json: { products: products }
+  end
+
+  def show
+    product = Product.find(params['id'])
+    render json: { product: product }
+  end
```

Comment:
So judging by user story and PR title you need to provide a list of products in this feature. Show method returns individual product so it is out of scope on this one, you should have a separate PR for that. The smaller the PRs the better.

Reply:
Noted, we rewrote show to return all products.

Reply:
Thomas asks to keep index method in and keep its functionality of returning an index of products.